packages/components/pie-button/src/button.scss

```diff
@@ -253,6 +241,14 @@ $breakpoint-wide: 768px;
     }
 }
 
+// The inner span for the button text is needed to handle the text-overflow
+// Without this, single word overflow wouldn't be possible in combination
+// with display: flex.
+.o-btn-text {
+    text-overflow: ellipsis;
```

Comment:
Do you mean for the ellipsis for single words?

This truncation only works when the button is set to be full-width. Otherwise, due to the way overflows work in general, the button will just keep increasing in size so no overflow can happen. Happy to show you on a call though, as it's a bit fiddly to test!